backend.tf

```diff
@@ -0,0 +1,25 @@
+# Storing terraform state in the cloud
+terraform {
+  backend "s3" {
+    bucket                      = "XXXXXXXX"
+    key                         = "terraform_module.tfstate"
+    region                      = "fr-par"
+    endpoint                    = "https://s3.fr-par.scw.cloud"
+    skip_credentials_validation = true
+    skip_region_validation      = true
+  }
+}
+
+
+/*
+
+For the credentials part:
```

Comment:
Maybe add this to the readme, in the pre-requisites part.
Or better, add a link to the official documentation in the readme
more traffic to our documentation is better for google visibility